tests/e2e/vmgroups_test.go

```diff
@@ -111,6 +111,9 @@ func (vg *VmGroupTest) TearDownSuite(c *C) {
 	out, err = ssh.InvokeCommand(vg.config.EsxHost, cmd)
 	log.Printf(out)
 
+	// Remove Config DB
+	adminutils.ConfigRemove(vg.config.EsxHost)
+
```

Comment:
This must not be done, its creating a fresh and clean plate for the next test to run and we may potentially wipe out a scenario that may lead to tests failing and bugs getting exposed. In fact, for all the e2e tests create the config DB once and let the tests run thru without removng the config DB (except for those tests that test the config DB itself).